src/terms/Hamiltonian.jl

```diff
@@ -87,7 +87,7 @@ Base.:*(H::Hamiltonian, ψ) = mul!(deepcopy(ψ), H, ψ)
 # (eg the density ρ)
 @timing function energy_hamiltonian(basis::PlaneWaveBasis, ψ, occ; kwargs...)
     # it: index into terms, ik: index into kpoints
-    ene_ops_arr = [ene_ops(term, ψ, occ; kwargs...) for term in basis.terms]
+    @timing "ene_ops" ene_ops_arr = [ene_ops(term, ψ, occ; kwargs...) for term in basis.terms]
```

Comment:
Should be consistent with https://github.com/JuliaMolSim/DFTK.jl/blob/master/src/terms/local.jl#L98. One way or the other ... I don't care.

Reply:
I don't understand what you mean here

Reply:
Ah sorry ... wrong annotation. I meant the individual `ene_ops` functions for each term where you use a different convention than for the timing label to the respective force terms.